.travis.yml

```diff
@@ -21,7 +21,7 @@ before_install:
   - JDK_BUILD_VERSION="$JDK_VERSION-b15"
   - JDK_FILE="jdk-$JDK_VERSION-$JDK_OS-$JDK_ARCH.tar.gz"
   - JDK_URL="https://people.cs.umass.edu/~rjust/defects4j/download/$JDK_FILE"
```

Comment:
The certificates for defects4j.org are now properly installed; let's change this url to: `https://defects4j.org/downloads/$JDK_FILE`.